scripts/checkin_tests.py

```diff
@@ -189,6 +200,33 @@ def verify_no_deny_list_words(file_contents, file_location):
             error_count += 1
     return error_count
 
+def verify_sample_files(root_path):
+    """Verify sample files meet the requirements and have not moved."""
+    sample_files_folder = os.path.join(root_path, "resources/sample_files")
+    media_folder = ".sample_media"
+    error_count = 0
+    file_list = []
+    for path, dirs, files in os.walk(sample_files_folder, topdown=True):
+        for file in files:
+            file_list.append(file)
+            file_path = os.path.join(path, file)
+            ext = os.path.splitext(file)[1].lstrip('.')
+            if file not in EXPECTED_SAMPLE_FILES:
+                logger.error(f"File '%s' in %s was not found in the list of expected sample files. If this is a new sample file, add it to the EXPECTED_SAMPLE_FILES list in checkin_tests.py.", file, sample_files_folder)
+                error_count += 1
+            if ext.lower() in MEDIA_FILE_TYPES:
+                if media_folder not in file_path:
+                    logger.error(f"File '%s' in %s must be in the %s directory.", file, sample_files_folder, media_folder)
+                    error_count += 1
+            if (os.path.getsize(file_path)/1000000) > 10:
+                logger.error(f"File '%s' in %s is larger than the allowed size for a sample file.", file, sample_files_folder)
+                error_count += 1
+
+    for sample_file in EXPECTED_SAMPLE_FILES:
```

Comment:
What's different from this and the above?
```
for file in files:
            ...
            if file not in EXPECTED_SAMPLE_FILES:
```

Reply:
One check is that nothing is in the folders that isn't in the list, and the other check is the reverse - checking that nothing in the list is missing from the folder. Two loops seemed the most straightforward way to do that.